### DISCUSSION TITLE
Spatial thermal comfort

### DISCUSSION THREAD

**Discussion by MajaWadling** [UMEP-dev/UMEP#605] [Q&A]
Hi,

When I try to use the "Tmrt_average" from SOLWEIG in the spatial thermal comfort tool to get a PET map it says this:

File"C:\Users/User/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\processing_umep\postprocessor\spatialtc_algorithm.py", line 274, in processAlgorithm
yyyyTmrt = int(filepath_tmrt[-18:-14])
ValueError: invalid literal for int() with base 10: '0/Tm'

But when I use a specific time ("Tmrt_2019_206_1200D") it works perfectly fine. Does anyone know what the problem is?

**Comment by biglimp** [comment]
Spatial Thermal Comfort requires a time specific Tmrt raster since all thermal comfort models also make use of meteorological data, e.g. air temperature. Tmrt_average cannot be used.

**Reply by MajaWadling** [reply]
Okay, thank you!
